docs/buildah-commit.md

```diff
@@ -76,6 +76,10 @@ Squash all of the new image's layers (including those inherited from a base imag
 
 Require HTTPS and verify certificates when talking to container registries (defaults to true)
 
+**--omit-timestamp** *bool-value*
+
+Set createdBy timestamp to epoch 0 to allow for deterministic builds (defaults to false)
```

Comment:
Perhaps:

Set the create timestamp to epoch 0 to allow for deterministic builds (defaults to false).  By default, the created timestamp is changed and written into the image manifest with every commit, causing the image's sha256 hash to be different even if the sources are exactly the same otherwise.  When `--omit-timestamp` is set to true, the created timestamp is always set to the epoch and therefore not changed, allowing the image's sha256 to remain the same."

Feel free to change the above, I've a bit of Saturday morning brain.

Reply:
done, your description is great and i just replaced mine with it. thanks!